target/linux/realtek/files-5.15/drivers/net/dsa/rtl83xx/dsa.c

```diff
@@ -1686,8 +1686,8 @@ static int rtl83xx_port_fdb_del(struct dsa_switch *ds, int port,
 
 	idx = rtl83xx_find_l2_hash_entry(priv, seed, true, &e);
 
-	pr_info("Found entry index %d, key %d and bucket %d\n", idx, idx >> 2, idx & 3);
 	if (idx >= 0) {
+		pr_info("Found entry index %d, key %d and bucket %d\n", idx, idx >> 2, idx & 3);
```

Comment:
This is a good change, and I squashed it into my MR :) which could allow you to drop this commit